### ISSUE TITLE
Why do you divide learning rate by batch size?

### ISSUE THREAD

**Issue by ljpadam** [TencentYoutuResearch/ObjectDetection-OneStageDet#36]
Usually, we increase lr as increasing batch size.

But, in _voc_train.py

line 66: optim = torch.optim.SGD(net.parameters(), lr=learning_rate/batch, momentum=momentum, dampening=0, weight_decay=decay*batch)

line 101: self.add_rate('learning_rate', lr_steps, [lr/self.batch_size for lr in lr_rates])

Why did you decide to decrease lr when use large batch size?

**Comment by mileistone** [comment]
be consistent to darknet
